spdx-utils/src/main/kotlin/model/SpdxExternalReference.kt

```diff
@@ -62,6 +62,21 @@ class SpdxExternalReference(
         OTHER("OTHER");
     }
 
+    enum class Type(
```

Comment:
Please remove the TODO in line 51, and change the type from String to Type in line 53.

Reply:
Removed the TODO but what do you mean with `String to Type in line 53`?

Reply:
We can not change `String` to `Type` as `referenceType` also can have free-from string besides the pre-defined values see https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/SPDXYAMLExample-2.2.spdx.yaml#L174

Reply:
We could probably solve that via a [sealed class](https://kotlinlang.org/docs/reference/sealed-classes.html).